src/ifp/src/InitFloorplan.cc

```diff
@@ -296,8 +296,8 @@ InitFloorplan::initFloorplan(double die_lx,
       uint site_dx = site->getWidth();
       uint site_dy = site->getHeight();
       // floor core lower left corner to multiple of site dx/dy.
-      int clx = (metersToMfgGrid(core_lx) / site_dx) * site_dx;
-      int cly = (metersToMfgGrid(core_ly) / site_dy) * site_dy;
```

Comment:
Is this change just allowing the core dimensions to exist without being tied to a site multiple? This seems like a good change in terms of saving area. Are there any other benefits? Just trying to understand the intention of the change.

Reply:
It fixes a bug. There is no reason that I am aware of that the lower left corner is a multiple of the site.